python/prompt-management/temp.py

```
@@ -0,0 +1,160 @@
+    escaped = escaped.replace('\t', '\\t')
+    escaped = escaped.replace('\0', '\\0')
+    escaped = escaped.replace('\b', '\\b')
+    escaped = escaped.replace('\f', '\\f')
+    
+    # Replace any remaining problematic characters
+    escaped = re.sub(r'[\x00-\x1f\x7f-\x9f]', '', escaped)  # Remove control chars
+    
+    return f"'{escaped}'"
+
+
+def generate_clickhouse_inserts(df, table_name, use_format_function=False):
```

Ruff (0.11.9)

Missing return type annotation for public function `generate_clickhouse_inserts`

(ANN201)